packages/shared/providers.yaml

```diff
@@ -3700,6 +3700,31 @@ microsoft-power-bi:
         base_url: https://api.powerbi.com
     docs: https://docs.nango.dev/integrations/all/microsoft-power-bi
 
+mindbody:
+    display_name: Mindbody
```

Comment:
Since Mindybody supports more than one auth, you can have this for the API key.

```
mindbody-api-key:
    display_name: Mindbody (api key)

Reply:
renamed to `Mindbody (api key)`